src/openMVG/sfm/pipelines/global/GlobalSfM_translation_averaging.cpp

```diff
@@ -633,13 +633,15 @@ bool GlobalSfM_Translation_AveragingSolver::Estimate_T_triplet(
   Mat* xxx[3] = {&x1, &x2, &x3};
   std::set<IndexT> intrinsic_ids;
   size_t cpt = 0;
-  for (tracks::STLMAPTracks::const_iterator iterTracks = tracks.begin();
-    iterTracks != tracks.end(); ++iterTracks, ++cpt) {
-    const tracks::submapTrack & subTrack = iterTracks->second;
+  for (tracks::TracksMap::const_iterator iterTracks = tracks.begin();
+    iterTracks != tracks.end(); ++iterTracks, ++cpt)
+  {
+    const tracks::Track & track = iterTracks->second;
     size_t index = 0;
-    for (tracks::submapTrack::const_iterator iter = subTrack.begin(); iter != subTrack.end(); ++iter, ++index) {
+    for (tracks::Track::FeatureIdPerView::const_iterator iter = track.featPerView.begin(); iter != track.featPerView.end(); ++iter, ++index)
```

Comment:
Same.
We don't use 'auto' here in order to be explicit and more understandable with low level type